commands/release.go

```diff
@@ -195,6 +205,41 @@ func showRelease(cmd *Command, args *Args) {
 	os.Exit(0)
 }
 
+func downloadRelease(cmd *Command, args *Args) {
+	tagName := cmd.Arg(0)
+	if tagName == "" {
+		utils.Check(fmt.Errorf("Missing argument TAG"))
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	release, err := gh.FetchRelease(project, tagName)
+	utils.Check(err)
+
+	for _, asset := range release.Assets {
+		assetReader, apiError := gh.DownloadReleaseAsset(asset.ApiUrl)
+		utils.Check(apiError)
+		defer assetReader.Close()
+
+		ui.Printf("Downloading %s ...\n", asset.Name)
+
+		assetFile, err := os.OpenFile(asset.Name, os.O_WRONLY|os.O_CREATE|os.O_EXCL, 0644)
+		utils.Check(err)
+		defer assetFile.Close()
+
+		_, err = io.Copy(assetFile, assetReader)
+		utils.Check(err)
+	}
+
+	os.Exit(0)
```

Comment:
I believe so. Without this line it seems to fall through to a missing argument handler and tests fail. I didn't look into this deeply, just a guess.